common/extensions/api/_api_features.json

```diff
@@ -2,7 +2,8 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 // Command to get whitelist ID: echo -n mnojpmjdmbbfmejpflffifhffcmidifd | openssl sha1 | tr '[:lower:]' '[:upper:]'
-// Same for sync api:           echo -n nomlkjnggnifocmealianaaiobmebgil | openssl sha1
+// Same for sync api:           echo -n nomlkjnggnifocmealianaaiobmebgil | openssl sha1 | tr '[:lower:]' '[:upper:]'
+// Same for ipfs-companion:     echo -n nibjojkomfdiaoajekhjakgkdhaomnch | openssl sha1 | tr '[:lower:]' '[:upper:]'
```

Comment:
`780bf954c0f7c586ea9662d4f967771f49cc2114` is the sha1 of `nibjojkomfdiaoajekhjakgkdhaomnch`; what does `FF32507DC3DB5DFFD1D6733187C84D4B74713D63` correspond to?

Reply:
IPFS beta extension, it's the 2nd commit in the PR.